make.js

```diff
@@ -0,0 +1,17 @@
+function make(...arr) {
```

Comment:
Very good solution, especially 'partial application'!

Mark for this function is 20.